packages/cursorless-vscode/src/extension.ts

```diff
@@ -76,25 +77,29 @@ export async function activate(
   const parseTreeApi = await getParseTreeApi();
 
   const { vscodeIDE, hats, fileSystem } = await createVscodeIde(context);
+  const isTesting = vscodeIDE.runMode === "test";
```

Comment:
We're doing this exact check in multiple places I think storing it to a variable increases readability